src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,77 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racinggame.dto.CarDto;
+import racinggame.dto.SnapShotDto;
+import racinggame.random.Radom;
+
+public class Cars {
+
+    private static final int DEFAULT_POSITION = 1;
+    private final SnapShotStore snapShotStore = new SnapShotStore();
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if(cars.isEmpty())throw new IllegalArgumentException("자동차가 비여 있을 수 없습니다.");
+        this.cars = cars;
+    }
+
+    public static Cars of(int count) {
+        return new Cars(carInitialization(count));
+    }
+
+    private static ArrayList<Car> carInitialization(int carCount) {
+        ArrayList<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carCount; i++) {
+            cars.add(new Car(DEFAULT_POSITION));
+        }
+        return cars;
+    }
+
+    public int count() {
+        return cars.size();
+    }
+
+    public void move(Radom radom) {
+        for(int i=0;i<count();i++){
+            move(i,radom.generate());
+        }
+        snapShotStore.save(nowState());
+    }
```

Comment:
```suggestion
    public void move(Radom radom) {
        for (Car car : cars) {
            car.move(radom.generate());
        }
        snapShotStore.save(nowState());
    }
```
향상된 for-loop을 사용해서 단순화 시켜볼 수 있지 않을까요?